pages/1.11/networking/SDN/dcos-overlay/index.md

```diff
@@ -0,0 +1,350 @@
+* You can run applications that require intra-cluster connectivity, like Cassandra, HDFS, and Riak.
+* You can create multiple virtual networks to isolate different portions of your organization, for instance, development, marketing, and production.
+
+You can learn more about the design and implementation of DC/OS overlay [here](/1.11/overview/design/overlay).
+
+The default configuration of DC/OS overlay provides an IPv4 virtual network, `dcos`, and an IPv6 virtual network `dcos6` whose YAML configuration is as follows:
+
+```yaml
+ dcos_overlay_network :
+   vtep_subnet: 44.128.0.0/20
+   vtep_subnet6: fd01:a::/64
+   vtep_mac_oui: 70:B3:D5:00:00:00
+   overlays:
+     - name: dcos
+       subnet: 9.0.0.0/8
+       prefix: 24
+     - name: dcos6
+       subnet6: fd01:b::/64
+       prefix6: 80
+```
+
+Each virtual network is identified by a canonical `name` (see [limitations](#limitations) for constraints on naming virtual networks). Containers launched on a virtual network get an IP address from the subnet allocated to the virtual network. To remove the dependency on a global IPAM, the overlay subnet is further split into smaller subnets. Each of the smaller subnets is allocated to an agent. The agents can then use a host-local IPAM to allocate IP addresses from their respective subnets to containers launched on the agent and attached to the given
+overlay. The `prefix` determines the size of the subnet (carved from the overlay subnet) allocated to each agent and thus defines the number of agents on which the overlay can run. E.g., in the default configuration above the virtual network `dcos` is allocated a /8 subnet (in the “subnet” field), which is then divided into /26 container subnets to be used on each host that will be part of the network (in the “prefix” field) as shown:
+
+![Virtual network address space](/1.11/img/overlay-network-address-space.png)
+
+The bits reserved for ContainerID (6 in this example) are then subdivided into two equal groups (of 5 bits in this example) that are used for Mesos containers and Docker containers respectively. With the default configuration, each agent will be able to host a maximum of 2^5=32 Mesos containers and 32 docker containers. With this specific configuration, if a service tries to launch more than 32 tasks on the Mesos containerizer or the Docker containerizer, it will receive a `TASK_FAILED`. Consult the [limitations](#limitations) section of the main Virtual Networks page to learn more about this constraint.
+
+While the above example is specifically for an IPv4 virtual network, the same logic can be applied to the IPv6 virtual network `dcos6` as well. The only caveat being that currently IPv6 is supported only for Docker containers. (*NOTE: If you try launching a UCR container on `dcos6`, it will result in a container launch failure.)* 
```

Comment:
s/(*NOTE: If you try launching a UCR container on `dcos6`, it will result in a container launch failure.)*/(*NOTE: If you try launching a UCR container on `dcos6`, it will result in a container launch failure.*)